### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by lindig**
Explain the 3 name spaces: upstream, xs, xs-extra.

Signed-off-by: Christian <christian.lindig@citrix.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,33 +13,23 @@ remote Opam repository:
...
-    +-- README.md
+Packages are organised into namespaces for organisational aspects. They
+are equivalent from the perspective of the Opam user.
+
+* `upstream`: packages that we don't control with fixed versions.
+* `xs`: packages that we control, with fixed versions.
+* `xs-extra`: packages that we control, following their respective
+  master branch.
+
+    ./packages
```
Comment (gaborigloi) [inline README.md]: Looking at https://github.com/lindig/xs-opam/, this is not recognized as a code block sadly.

File: README.md [hunk 2]
```diff
@@ -13,33 +13,23 @@ remote Opam repository:
...
-    |       |   +-- descr
-    |       |   +-- opam
-    |       |   \-- url
-    |       \-- xenstore_transport.0.9.4
-    |           +-- descr
-    |           +-- findlib
-    |           +-- opam
-    |           \-- url
-    +-- README.md
+Packages are organised into namespaces for organisational aspects. They
```
Comment (gaborigloi) [inline README.md]: Would it be better to move this paragraph with the bullet points to the bottom, below the layout and general description of `packages/`? Maybe that would improve the flow of the text.

File: README.md (hunk 1 above)
Comment (mseri) [inline README.md]: You are right! I missed it. We need to add 4 more spaces.

File: README.md (hunk 2 above)
Comment (mseri) [inline README.md]: I don't know. I think it makes sense to have the information straight-away. But I would be happy either way

File: README.md (hunk 1 above)
Comment (lindig) [inline README.md]: I've fixed the layout problem and added some small touches.

**Pull request merged**